.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: github-project-issue-to-sheets
+
+# Controls when the action will run. Triggers the workflow on push or pull request
```

Comment:
The comment is incorrect and not needed IMO.

Reply:
smh this is why I shouldn't be allowed near a pastebuffer